Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs

```diff
@@ -141,7 +144,7 @@ private static object WriteRowToJson(DataTable dt, int i, Options options)
 
```

Comment:
https://github.com/FrendsPlatform/Frends.Excel/blob/4b47382fa662be9dea393a463b601ff6edfc1922/Frends.Excel.ConvertToJSON/Frends.Excel.ConvertToJSON/Excel.cs#L142
This can be a one-line if statement.